packages/main/src/Calendar.ts

```diff
@@ -441,15 +443,15 @@ class Calendar extends CalendarPart {
 	}
 
 	onSelectedDatesChange(e: CustomEvent<DayPickerChangeEventDetail>) {
-		const timestamp = e.detail.timestamp;
```

Comment:
Why were these changes necessary? To assign to `this.timestamp` early, to rename the local variable from `ts` to `timestamp`, etc...? This only looks like refactoring?

Reply:
@tsanislavgatev didn't like the ts, he commented that ts could many things, and to make it timestamp instead.